recipes/drupal8.php

```diff
@@ -20,3 +20,11 @@
   // https://www.drush.org/latest/deploycommand/
   invoke('drush:deploy');
 })->once();
+
+task('drush:config:export', function() {
+  $releasesList = get('releases_list');
+  $prev_release = max($releasesList);
+  $destination = '{{deploy_path}}/releases/' . $prev_release . '/config/backup';
+  run("mkdir $destination");
```

Comment:
It's worth it to add a `-p` flag to ensure all directories exist in the `mkdir` command. You will see Deployer doing this all over it's code.